Index: 5.66-r0/sysroot-destdir/lib/systemd/system/bluetooth.service
===================================================================
--- 5.66-r0.orig/sysroot-destdir/lib/systemd/system/bluetooth.service	2025-10-21 13:16:07.045083439 +0530
+++ 5.66-r0/sysroot-destdir/lib/systemd/system/bluetooth.service	2025-10-23 14:30:01.480589274 +0530
@@ -9,7 +9,7 @@
 Type=dbus
 BusName=org.bluez
 ExecStartPre=/bin/sleep 5
-ExecStart=/usr/lib/bluez5/bluetooth/bluetoothd -d
+ExecStart=/usr/lib/bluez5/bluetooth/bluetoothd -n -E -d
 NotifyAccess=main
 #WatchdogSec=10
 #Restart=on-failure
